README.md

```diff
@@ -23,3 +23,11 @@ Use `git submodule init` and `git submodule update` to init and update the conte
 ```sh
 docker-compose up
 ```
+
+## Tests auto
```

Comment:
"Préparer la db de test" plutôt ?

Reply:
ben en fait il me semble que c'est fait automatiquement

> In order to run your tests, your test database will need to have the current structure. The test helper checks whether your test database has any pending migrations. It will try to load your db/schema.rb or db/structure.sql into the test database. 

https://github.com/rails/rails/blob/main/activerecord/lib/active_record/migration.rb#L711-L731

Reply:
Comme chez moi j'ai dû faire la manip, je me suis dit que ça serait bien pour le prochain de l'avoir noté dans le Readme. Mais si tu penses que c'est pas pertinent, on peut le virer.

Reply:
j'ai mis "Si la db de test n'est pas automatiquement créée"